src/read_pool.rs

```diff
@@ -106,7 +104,7 @@ pub enum ReadPoolHandle {
         read_pool_low: FuturePool,
     },
     Yatp {
-        remote: Remote<TaskCell>,
+        remote: FuturePool,
```

Comment:
FuturePool already implement running tasks statistics, scaling tasks, so you should remove the implement in yatp handle and use FuturePool's directly.

Reply:
The metric `running_task` is still being saved for compatibility reasons.

Reply:
There is a metric with the same function in FuturePoolInner.env called `metrics_running_task_count`, you can use it to replace the running tasks in Yatp handle.

Reply:
Ok. Then I think you can at least remove `max_tasks` and `pool_size` as they are used for scaling pool size. BTW, please add a deprecated comment for `running_tasks` so we may permanently remove it in v8.0.